src/spectrogramplot.h

```diff
@@ -62,6 +67,7 @@ public slots:
     static const int tileSize = 65536; // This must be a multiple of the maximum FFT size
 
     std::shared_ptr<SampleSource<std::complex<float>>> inputSource;
+    std::vector<AnnotationLocation> annotationLocations;
```

Comment:
Suggestion: Change the name to `visibleAnnotationLocations`. That makes it clearer that this is an ephemeral list.

Reply:
I have applied this suggestion.